### PR TITLE
update for cedar#921

### PR DESCRIPTION
**Pull request by cdisselkoen**
*Issue #, if available:*

cedar-policy/cedar#921

### CODE REVIEW

**Review by shaobo-he-aws — APPROVED** [review]

**Review by khieta — APPROVED** [review]

File: CedarJavaFFI/src/objects.rs [hunk 1]
```diff
@@ -235,7 +235,7 @@ impl<'a> JEntityId<'a> {
 
     /// Decode the object into its string representation
     pub fn get_string_repr(&self) -> String {
-        self.id.to_string()
+        self.id.as_ref().to_string()
```
Comment (khieta) [inline CedarJavaFFI/src/objects.rs:238]: Why not `.escaped()`? That would have been the original behavior, right?

Comment (cdisselkoen) [reply CedarJavaFFI/src/objects.rs:238]: True, and I can change it.  I thought that perhaps folks manipulating the string in Java might expect to manipulate the unescaped string.  But I suppose no one has complained about the current behavior

Comment (khieta) [reply CedarJavaFFI/src/objects.rs:238]: No strong preference, I was just wondering if the behavior change was intentional. I think both options are reasonable, but might be better stick with the status quo 🤷‍♀️

Comment (cdisselkoen) [reply CedarJavaFFI/src/objects.rs]: Changed to `.escaped()`

**Review by shaobo-he-aws — APPROVED** [review]

**Pull request merged**